recipes/distinctipy/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "distinctipy" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/distinctipy-{{ version }}.tar.gz
+  sha256: 8145a422766a7e653549d8bec2978c0b8225f91a6d08bece46228c44a5ec962b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
```

```suggestion
    - numpy >=1.16.3
```

Ref: https://github.com/alan-turing-institute/distinctipy/blob/9f9f15e746884be03292e9f8398c7c2a98cc19e4/requirements/runtime.txt#L1

---

Thanks! Grayskull missed that :-/